frontend/express/views/forgot.html

```diff
@@ -135,6 +135,17 @@ <h1 style="font-weight: 500; text-align: center; line-height: 48px;" id="header"
 							}
 							else {
 								$('#message').html($.i18n.map['forgot.result']);
+								var prepareEmail = $('#remember-password-email').val();
+								prepareEmail = prepareEmail.replace(/(\w{1})[\w.-]+@([\w.]+\w)/, "$1***@$2");
+								var emailmessage = `An email has been sent to your email address, ${prepareEmail}. Follow the directions in the email to reset your password.`
```

Comment:
We don't use string interpolation on UI, so please convert this to "..."  + prepareEmail + "..." format and remove backticks (`) :slightly_smiling_face: